docs/source/jobs.rst

```diff
@@ -239,41 +239,137 @@ Use cases for the **job document** include, but are not limited to:
 
 Job Data Storage
 ================
-
-Basics
-------
-
-Large numerical or text data can be stored in the :py:attr:`Job.data` container, which is an instance of :class:`signac.H5Store`.
+Large numerical or text data can be stored in the :py:attr:`Job.data` container, which is an instance of :class:`signac.H5Store`. 
 This container uses a file in `HDF5`_ format to store array-like or dictionary-like information.
 Like the :py:attr:`Job.document`, this information can be accessed using key-value pairs.
 Unlike the :py:attr:`Job.document`, :attr:`Job.data` is not searchable.
-
 .. _`HDF5`: https://portal.hdfgroup.org/display/HDF5/HDF5
 
+Job associated data may be stored through :py:attr:`Job.data` or :py:attr:`Job.stores`.
+This section will focus on examples and usage of :py:attr:`Job.data`.
+Data written with :py:attr:`Job.data` is stored in a file named ``signac_data.h5`` in the associated `job` folder.
+Data written with :py:attr:`Job.stores['key_name']` is stored in a file named ``key_name.h5``.
+For cases where `job`-associated data may be accessed from multiple sources at the same time or other instances where multiple files may be preferred to one large file, :py:attr:`Job.stores` should be used instead of :py:attr:`Job.data`.
+Further discussion of :py:attr:`Job.stores` is provided in the folling section, Job Stores.
+
+Reading and Writing data
+-----------------------
```

Comment:
```suggestion
------------------------
```

Reply:
Sphinx should actually complain about that.

Reply:
Yes, Sphinx will throw a warning about this. I recommend rendering the docs locally and looking at them to ensure the formatting is what you want and to check for warnings.